vhost_user_block/src/lib.rs

```diff
@@ -228,8 +228,8 @@ impl VhostUserBackend for VhostUserBlkBackend {
         VhostUserProtocolFeatures::CONFIG
     }
 
-    fn set_event_idx(&mut self, enabled: bool) {
-        self.event_idx = enabled;
+    fn set_event_idx(&mut self, _enabled: bool) {
+        self.event_idx = false;
```

Comment:
This is not a good idea. @slp do you have any idea on what's going on here?

Reply:
@rbradford That's wrong, it's inhibiting vhost_user_blk EVENT_IDX support.

Reply:
@slp My thoughts exactly! Perhaps you could lend a hand on understanding what's going wrong.

Reply:
@slp @rbradford  I comment event_idx just to verify if event_idx is related to the failure of test_boot_from_vhost_user_blk and test_vhost_user_blk_self_spawning. Yes, this is not a fix, but to circumvented the issue temporary.